packages/lesswrong/components/recentDiscussion/RecentDiscussionTag.tsx

```diff
@@ -104,40 +110,27 @@ const RecentDiscussionTag = ({ tag, refetch = () => {}, comments, expandAllThrea
     condensed: true,
   }
   
-  let metadataWording = ''
-  if (taggingNameIsSet.get()) {
-    if (tag.wikiOnly) {
-      metadataWording = `${taggingNameCapitalSetting.get()} page`
-    } else {
-      metadataWording = `${taggingNameCapitalSetting.get()} page - ${tag.postCount} posts`
-    }
-  } else {
-    if (tag.wikiOnly) {
-      metadataWording = `Wiki page`
-    } else {
-      metadataWording = `Tag page - ${tag.postCount} posts`
-    }
-  }
+  const metadataWording = tag.wikiOnly ? "Wiki page" : `${taggingNameCapitalSetting.get()} page - ${tag.postCount} posts`;
```

Comment:
Why did this logic change?

Reply:
I thought the initial logic didn't really make sense, and was also just overcomplicated, in practice the only change here is that wikiOnly pages will now say "Wiki page" rather than "Topic page". I could be convinced that this doesn't belong in this PR... I started changing it because I was going to make it say something different for subforums but then decided to just remove it instead